include/sys/zio.h

```diff
@@ -114,6 +115,22 @@ enum zio_compress {
 	ZIO_COMPRESS_GZIP_9,
 	ZIO_COMPRESS_ZLE,
 	ZIO_COMPRESS_LZ4,
+	ZIO_COMPRESS_LZ4HC_1,
```

Comment:
Unfortunately, this is not the best way to do it (GZIP_# is also not great).  I assume that (like gzip), any lz4hc_# can be decompressed by one algorithm.  Adding all these values eats into the limited namespace of BP_GET_COMPRESS() (128 values).  We should differentiate between the values that are set on the "compression" property (which could be lz4hc-2 or lz3hc-3, etc) and the values that are stored in the BP (where we would add at most one value).